src/quiz/quiz.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Controller, Get, Param } from "@nestjs/common";
+import { QuizService } from "./quiz.service";
+
+@Controller('/api')
+export class QuizController {
+  constructor(private readonly quizService: QuizService) {}
+
+  @Get('/quiz')
+  getQuiz(): string {
+    return this.quizService.getQuiz()
+  }
+
+  @Get('/quiz/:id')
```

Comment:
same here

Reply:
resolved by new PR (couldn't push anything to current branch)